grpcurl/plan.sh

```diff
@@ -1,31 +1,28 @@
 gopkg="github.com/fullstorydev/grpcurl"
 pkg_name=grpcurl
 pkg_origin=core
+pkg_version="1.5.1"
 pkg_description="Like cURL, but for gRPC: Command-line tool for interacting with gRPC servers"
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
-pkg_source="https://$gopkg/cmd/grpcurl"
+pkg_source="https://github.com/fullstorydev/grpcurl/archive/v${pkg_version}.tar.gz"
 pkg_upstream_url="https://$gopkg"
-pkg_version="1.3.0"
 pkg_license=('MIT')
 pkg_bin_dirs=(bin)
-pkg_scaffolding=core/scaffolding-go
-scaffolding_go_base_path=github.com/fullstorydev
+pkg_build_deps=(
+    core/git
+    core/go
+)
+pkg_shasum="0e046500122cb533f9565574a5b06fb74f5c97fe01c93b7550edd5e2edc953ce"
 
-do_download() {
-  # `-d`: don't let go build it, we'll have to build this ourselves
-  go get -d $gopkg
 
-  pushd "${scaffolding_go_gopath:?}/src/$gopkg"
-    git reset --hard "v$pkg_version"
-  popd
+do_install() {
+    echo "$pwd"
+    GO111MODULE=on CGO_ENABLED=0 GOARCH=amd64 GOOS=linux go build \
+      -ldflags '-X "main.version=dev build $(dev_build_version)"' \
```

Comment:
This line generates many questions for me: This is in single quotes so what is `$()` doing? Assuming it was in double quotes, where is dev_build_version defined? Why not use `$pkg_version` instead of `dev` for the version?

Reply:
copy paste... also what is dev_build_version

Reply:
```
[44][default:/src:2]# /hab/pkgs/jaym/grpcurl/1.5.1/20200428193630/bin/grpcurl -version
grpcurl dev build $(dev_build_version)
```

Reply:
fixed